Client.xcodeproj/project.pbxproj

```diff
@@ -20903,7 +20903,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/ecosia/ios-core.git";
 			requirement = {
-				branch = main;
+				branch = "MOB-2498_upgrade_braze";
```

Comment:
I guess this Core branch change was not meant for this PR?

Reply:
💯 - let's leave it for now and not merge it considering the following upcoming fixes.
Will point to `main` once we'll have the new `Core`